src/analytics/helpers.js

```diff
@@ -79,13 +80,19 @@ export const event = (eventName, value = {}, nonInteraction = false) => {
 }
 
 export const getDatasetOptionsChanges = (dataset, regeneratedDataset) => {
-  const changes = datasetOptionsKeys.map((key) => {
-    return `${getReadable(key, dataset[key])} -> ${getReadable(
-      key,
-      regeneratedDataset[key]
-    )}`
-  })
-  return changes.join('|')
+  const initial = datasetOptionsKeys
+    .map((key) => `${getReadable(key, dataset[key])}`)
+    .join('|')
+
+  if (regeneratedDataset) {
+    const changes = datasetOptionsKeys
+      .map((key) => `${getReadable(key, regeneratedDataset[key])}`)
+      .join('|')
+
+    return `${initial} -> ${changes}`
+  }
+
+  return initial
```

Comment:
I've revised this method to return the download options changes if any, otherwise returns the `initial` download options value.